pyrefinebio/high_level_functions.py

```diff
@@ -229,3 +228,38 @@ def download_quandfile_compendium(
         extract=extract,
         prompt=prompt
     )
+
+
+def create_token(silent=False):
+    """create_token
+
+    Automatically creates a Token, activates it, and stores it to the Config file.
+
+    By default, will prompt the user before activating and storing the created Token.
+
+    Returns:
+        Token
+
+    Parameters:
+        silent (bool): if true, you will not be prompted before activating and storing
+                       the Token.
+    """
+    if not silent:
+        print("Please review the refine.bio Terms of Use: https://www.refine.bio/terms and Privacy Policy: https://www.refine.bio/privacy")
+        yn = input("Do you understand and accept both documents? (Y/n)")
+
+        if yn.lower() in ("n", "no"):
```

Comment:
```suggestion
        if yn.lower() not in ("y", "yes"):
```
This way if someone types in "No way!" we don't take it as acceptance.